site/content/installation/installing-ngf/manifests.md

```diff
@@ -53,8 +53,22 @@ Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps.
 
 #### Stable release
 
+- For NGINX:
+
    ```shell
-   kubectl apply -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.0.0/nginx-gateway.yaml
+   kubectl apply -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.1.0/nginx-gateway.yaml
+   ```
+
+- For NGINX Plus:
+
+   Download the [deployment YAML](https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.2.0/nginx-plus-gateway.yaml).
+
+   Update the `nginx-plus-gateway.yaml` file to include your chosen NGINX Plus image from the F5 Container registry or your custom image.
+
+   {{< note >}}Ensure that you [Enable Usage Reporting]({{< relref "installation/usage-reporting.md" >}}) before installing.{{< /note >}}
```

Comment:
```suggestion
   {{< note >}}Ensure that you [Enable Usage Reporting]({{< relref "installation/usage-reporting.md" >}}) before applying.{{< /note >}}
```
Apply/Install are essentially synonyms in context but since the verb in the command is apply, I think it's best to match it.